torch/__init__.py

```diff
@@ -381,8 +381,11 @@ def use_deterministic_algorithms(d):
         * :class:`torch.nn.CTCLoss` when called on a CUDA tensor that requires grad
         * :class:`torch.nn.EmbeddingBag` when called on a CUDA tensor that requires grad
         * :func:`torch.scatter_add_` when called on a CUDA tensor
+        * :func:`torch.__getitem__` when `self` is a CPU tensor that requires
```

Comment:
Should this mention that it is only in the backward?
More generally, in this list, are you differentiating between forward and backward?

Reply:
Everything marked with "tensor that requires grad" is actually referring to the backward method. I think you're right, it would be better to be more explicit. Maybe just something like ":func:`torch.__getitem__` backward when `self` is a CPU tensor..."